src/garage/sampler/evaluation_sampler.py

```diff
@@ -0,0 +1,61 @@
+"""Sampler for evaluating off-policy algorithms."""
+
+from garage.sampler.base import BaseSampler
+from garage.sampler.utils import rollout
+
+
+class EvaluationSampler(BaseSampler):
```

Comment:
I doubt an `EvaluationSampler` external to algorithms can cover all situations. Some challenging examples have been explained in #1091 . So instead of an additional sampler, I propose to just expose `rollout()` to the algorithm. This function should be the barebone to do anything with sampling.

Reply:
This is why I proposed doing evaluation sampling in the algorithm. It can use whichever class it wants, including none.

I think Avnish's PR has a class called SimpleSampler which is basically equivalent to this one.

Reply:
It's just by our current design, algorithms don't know `env`. Is it a good practice to use `env` passed by `runner.train()`?

Reply:
So I added the stats, then tried to use `rollout` in algorithms directly, instead of using a `Sampler`. It seems better than before.